src/domains/messages-common/05_apim_v2.tf

```diff
@@ -188,8 +188,60 @@ data "azurerm_api_management_product" "apim_v2_product_services" {
   resource_group_name = data.azurerm_api_management.apim_v2_api.resource_group_name
 }
 
+# APIM APIs
+
+data "http" "messages_sending_external_openapi" {
+  url = "https://raw.githubusercontent.com/pagopa/io-functions-services-messages/master/openapi/index_external.yaml"
+}
+
+module "apim_v2_messages_sending_external_api_v1" {
+  source = "git::https://github.com/pagopa/terraform-azurerm-v3//api_management_api?ref=v7.69.1"
+
+  name                  = format("%s-messages-sending-external-api", local.product)
```

Comment:
In [this RFC](https://pagopa.atlassian.net/wiki/spaces/DevEx/pages/1032880258/RFC-DX-005+Naming+convention+nuove+risorse+Azure) you can find the newly introduced naming convention for azure resources. If you agree, it would be nice to employ it when naming these new resources.

Reply:
`messages-sending` is the name of the pointed functions for the `external` users, `api` is the resource type. We do not have a cardinality. So are we just missing the `weu`?

Reply:
That "instance-number" suffix is a two characters incremental number that may be useful when you'd like to replace the resource without having to change the actual resource name. In that case you would just change the suffix from `-01` to `-02`. 

If you agree, I would add it to fully follow the new naming convention